demo/server/src/modules/passport-auth/index.ts

```diff
@@ -4,45 +4,52 @@ import * as express from 'express';
 import { SessionOptions } from 'express-session';
 import * as jwt from 'jsonwebtoken';
 import * as logger from 'loglevel';
+import appConfig from '../../util/Config';
 
 // Implementation for fetching and mapping user data
 import DemoUserRepository, { SampleUserService } from './DemoUserRepository';
 
-export function init(app: express.Router, sessionOpts?: SessionOptions) {
+const config = appConfig.getConfig();
+
+export function init(router: express.Router, sessionOpts?: SessionOptions) {
   // Initialize user data repository and map current user
   const userRepo: UserRepository = new DemoUserRepository();
   const userService: UserService = new SampleUserService();
   const authService: PassportAuth = new PassportAuth(userRepo, userService);
-  authService.init(app, sessionOpts);
+
   if (sessionOpts) {
-      createPortalRoutes(app, authService, userRepo);
+    authService.init(router, sessionOpts);
+    createPortalRoutes(router, authService, userRepo);
   } else {
-    createMobileRoutes(app, authService, userRepo, userService);
+    authService.init(router, sessionOpts, config.jwtSecret);
+    createMobileRoutes(router, userRepo, userService);
   }
   return authService;
 }
 
 function createPortalRoutes(router: express.Router, authService: PassportAuth, userRepo: UserRepository) {
   getLogger().info('Creating Portal Routes');
   router.get('/login', (req: express.Request, res: express.Response) => {
-    if (req.session) {
-      req.session.returnTo = req.headers.referer;
-    }
     return res.render('login', {
       title: 'Portal Feedhenry Workforce Management'
     });
   });
 
-  router.post('/login', authService.authenticate('local', '/', '/loginError'));
+  router.post('/login', authService.authenticate('local', {
+    successReturnToOrRedirect: config.clientUrl.portal,
+    failureRedirect: '/loginError'
+  }));
 
   router.get('/loginError', (req: express.Request, res: express.Response) => {
     return res.render('login', {
-      title: 'Feedhenry Workforce Management',
+      title: 'Portal Feedhenry Workforce Management',
```

Comment:
[Style][Optional] `Portal Feedhenry Workforce Management` => `RainCatcher Portal`
[Optional] - Externalize to config. 
[Optional] - Move to passport (as they are now really connected to the actual strategies) not universal.